lms/static/coffee/src/instructor_dashboard/queries.coffee

```diff
@@ -0,0 +1,685 @@
+    if arr.length == 4
+      progressCell.innerHTML = $progress_icon[0].outerHTML
+    else
+      if arr[4] == null
+        progressCell.innerHTML = $broken_icon[0].outerHTML
+      else if arr[4] == true
+        progressCell.innerHTML = $done_icon[0].outerHTML
+        row.classList.remove('working')
+      else
+        progressCell.innerHTML = $progress_icon[0].outerHTML
+    $removeBtn = $(_.template('<div class="remove"><i class="icon-remove-sign">
+      </i> <%= label %></div>', {label: 'Remove'}))
+    $removeBtn.click (event) =>
+      targ = event.target
+      while (!targ.classList.contains('remove'))
+        targ = targ.parentNode
+      curRow = targ.parentNode.parentNode
+      curRow.remove()
+      if curRow.hasAttribute('query')
+        queryToDelete = curRow.getAttribute('query')
+        @delete_temp_query(queryToDelete)
+      @check_done()
+    $td = $ '<td>'
+    $td.append($removeBtn)
+    row.appendChild($td[0])
+    return $(row)
+
+  save_query: (cb)->
+    cur_queries = []
+    tab = $('.emailWidget.queryTableBody')
+    rows = tab.find('tr')
+    _.each rows, (row) ->
+      cur_queries.push(row.getAttribute('query'))
+    send_data =
+      existing: cur_queries.join(',')
+      savedName: $(".emailWidget.savequeryname").val()
+    $(".emailWidget.savequeryname").val("")
+    $.ajax(
+      type: 'POST'
+      dataType: 'json'
+      url: @$saveQueryBtn.data('endpoint')
+      data: send_data
+      success: (data) -> cb? null, data
+      error: std_ajax_err ->
+        cb? gettext('Error saving query')
+    )
+  # save queries in active queries
+  send_save_query: ->
+    @save_query (error, data) =>
+      if error
+        return @show_errors error
+      $('#send_email select').append('<option value="' + data['group_id'] + '">' + data['group_title'] + '</option>')
+      @load_saved_queries()
+
+  get_estimated: (cb)->
+    curQueries = []
+    tab = $('.emailWidget.queryTableBody')
+    rows = tab.find('tr')
+    _.each rows, (row) ->
+      curQueries.push(row.getAttribute('query'))
+    send_data =
+      existing: curQueries.join(',')
+    $.ajax
+      dataType: 'json'
+      url: @$totalEndpoint
+      data: send_data
+      success: (data) -> cb? null, data
+      error: std_ajax_err ->
+        cb? gettext('Error getting estimated')
+
+  # estimate the students selected
+  reload_estimated: ->
+    $('.emailWidget.estimated').html(gettext('Calculating'))
+    @get_estimated (error, students) =>
+      if students['success'] == false
+        $('.emailWidget.estimated').html(gettext('0 students selected'))
+        return
+      studentsList = students['data']
+      queryId = students['query_id']
+      # abort on error
+      if error
+        return @show_errors error
+      $numberStudents = studentsList.length
+      $('.emailWidget.estimated').html(gettext('approx ' + $numberStudents + ' students selected'))
+  # set error display
+  show_errors: (msg) -> @$error_section?.text msg
```

Comment:
When this section was still in membership, `@$error_section` was defined here: https://github.com/Stanford-Online/edx-platform/blob/master/lms/static/coffee/src/instructor_dashboard/membership.coffee#L1341
as `@$auth_list_errors`, which refers to this div: https://github.com/Stanford-Online/edx-platform/blob/master/lms/templates/instructor/instructor_dashboard_2/membership.html#L157

So you are going to have to define your own div to display error messages in queries.html and initialize it in your constructor.

And then in clear_display add a line to clear this section: `@$error_section?.text ''`